### PR TITLE
Feature/update and delete transactions

### PR DESCRIPTION
**Pull request by FredrikAugust**
Implemented callbacks for edit and delete. The things remaining are the following (@NicolajNiels1 is going to implement this):

- Proper interface for editing and deleting
- Error handling
- Validation on edit form

### CODE REVIEW

**Comment by FredrikAugust** [comment]
I also changed it to show more when you click the transaction as we decided to change up the intermediary calculator implementation, and thus it makes more sense to show more content instead of adding to that when clicking the tx.

If this is not wanted, I could revert it.

**Comment by FredrikAugust** [comment]
This features the basic functionality needed to use this, and thus I will stage it for review. Going to start working on the company onboarding functionality.

File: src/components/molecules/UpdateTransaction.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+        placeholder="Notes regarding the transaction"
+      >
+        Notes
+      </TextArea>
+      <OutlinedButton type="submit">Update transaction</OutlinedButton>
+    </form>
+  );
+};
+
+export default UpdateTransaction;
```
Comment (NicolajNiels1) [inline src/components/molecules/UpdateTransaction.tsx]: I suggest name change to "EditTransaction"

File: src/store/reducers/company.ts [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import * as API from '../../mitochondria';
 
 type ICompany = import('../../declarations/company').ICompany;
+type ICompanyUser = import('../../declarations/company').ICompanyUser;
```
Comment (NicolajNiels1) [inline src/store/reducers/company.ts]: 🙌

**Comment by NicolajNiels1** [comment]
Shouldn't this be merged with master in stead?

Also, seems like #122 depends on this PR 🙂

**Comment by FredrikAugust** [comment]
> Shouldn't this be merged with master in stead?
> 
> Also, seems like #122 depends on this PR

It says it's supposed to be merged into feature/reducers because that was a dep when I began working on this, and it was not merged into master.

File: src/components/molecules/UpdateTransaction.tsx (hunk 1 above)
Comment (FredrikAugust) [reply src/components/molecules/UpdateTransaction.tsx]: Fixed  in `b699631`

**Pull request merged**